carto/do_subscriptions.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""
+Module for working with Data Observatory Subscriptions
+
+.. module:: carto.do_subscriptions
+   :platform: Unix, Windows
+   :synopsis: Module for working with Data Observatory Subscriptions
+
+.. moduleauthor:: Jesús Arroyo <jarroyo@carto.com>
+
+
+"""
+
+from pyrestcli.fields import CharField, FloatField
+
+from .resources import Resource, Manager
+from .paginators import CartoPaginator
+from .do_subscriptions_info import DOSubscriptionInfo
+
+
+API_VERSION = "v4"
+API_ENDPOINT = "api/{api_version}/do/subscriptions"
+
+
+class DOSubscription(Resource):
+    """
+    Represents a Data Observatory Subscriptions in CARTO.
+
+    """
+    dataset = CharField()
+    id = CharField()
+    project = CharField()
+    table = CharField()
+    type = CharField()
+
+    class Meta:
+        collection_endpoint = API_ENDPOINT.format(api_version=API_VERSION)
+        name_field = "id"
+
+
+class DOSubscriptionManager(Manager):
```

Comment:
I'd merge this as it is, but since this is not a CRUD API, we should override [all the Manager methods](https://github.com/danicarrion/pyrestcli/blob/319a0d9574f47a2ff23cbce197bf412f5000d6fa/pyrestcli/resources.py#L243-L344) except `all` and `pass` them or raise an error.

Same for the rest of the managers.

I'm creating a separate issue to fix this, and the other FIXME in this PR.

Reply:
https://github.com/CartoDB/carto-python/issues/152